docs/source/tutorials/intro.ipynb

```diff
@@ -294,14 +294,16 @@
     "# Plot the results\n",
```

Comment:
<p>Actually, the runner calls <code>suggest_experiments</code> , so it would be better to say that and not add the extra line of code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sustainable-processes/summit/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Just addressed this: I reverted the addition and expanded a bit the documentation text following this code.